src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,79 +1,72 @@
 package techcourse.myblog.domain;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 import java.util.Objects;
 
+@Entity
 public class Article {
     private static final String DEFAULT_COVER_URL = "/images/pages/index/study.jpg";
     private static final int NO_COVER_URL = 0;
-    private static long NEXT_ID = 1L;
 
-    private long articleId;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
     private String title;
-    private String contents;
     private String coverUrl;
+    private String contents;
 
     public Article() {
```

Comment:
JPA가 필요로 하는 기본 생성자는 private으로 만들어야 다른 개발자의 오용을 방지할 수 있어요. 아래의 글은 가볍게 참고만 해주세요.
https://goodyhlee.wordpress.com/2016/06/20/177

```suggestion
    private Article() {
```